internal/cli/serverless/sqluser/create.go

```diff
@@ -0,0 +1,302 @@
+// Copyright 2024 PingCAP, Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//      http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sqluser
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"tidbcloud-cli/internal"
+	"tidbcloud-cli/internal/config"
+	"tidbcloud-cli/internal/flag"
+	"tidbcloud-cli/internal/service/cloud"
+	"tidbcloud-cli/internal/telemetry"
+	"tidbcloud-cli/internal/ui"
+	"tidbcloud-cli/internal/util"
+
+	iamApi "tidbcloud-cli/pkg/tidbcloud/v1beta1/iam/client/account"
+	iamModel "tidbcloud-cli/pkg/tidbcloud/v1beta1/iam/models"
+	serverlessApi "tidbcloud-cli/pkg/tidbcloud/v1beta1/serverless/client/serverless_service"
+
+	"github.com/charmbracelet/bubbles/textinput"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/fatih/color"
+	"github.com/juju/errors"
+	"github.com/spf13/cobra"
+)
+
+type CreateOpts struct {
+	interactive bool
+}
+
+const (
+	WaitInterval = 5 * time.Second
+	WaitTimeout  = 2 * time.Minute
+)
+
+var createSQLUserField = map[string]int{
```

Comment:
Why not use enum?

Reply:
Using an enum is also OK, just referencing other implementations